client/control/tests/test_routes.js

```diff
@@ -1,8 +1,9 @@
 import { replace } from 'react-router-redux';
 import { shallow } from 'enzyme';
 
-import { createApp } from '../app.js';
-import NoMatch from '../components/NoMatch.js';
+import { createApp } from 'control/app';
+
+import NoMatch from 'control/components/NoMatch.js';
```

Comment:
This is a super tiny nit, but why the extra space here? Or is there a pattern here that I haven't seen yet?